spacy_llm/tasks/entity_linking.py

```diff
@@ -0,0 +1,310 @@
+import csv
```

Comment:
We'll have to refactor this one to be consistent with https://github.com/explosion/spacy-llm/pull/242

Reply:
And rename to `tasks/entity_linker.py` to be consistent with spaCy's naming.

Reply:
Done in https://github.com/explosion/spacy-llm/pull/241/commits/937c04d90e2e4f4181d7476f9c2ad8676b052e0e.